packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -180,12 +186,14 @@ class Auth {
     private silenceWarnings: boolean
     private logger: Logger
     private defaultDnt: boolean | undefined
-
+    private dntCookieName: AuthDataKeys
     constructor(config: AuthConfig) {
         // Special endpoint for injecting SLAS private client secret.
         const baseUrl = config.proxy.split(MOBIFY_PATH)[0]
         const privateClientEndpoint = `${baseUrl}${SLAS_PRIVATE_PROXY_PATH}`
-
+        this.dntCookieName = 'dw_dnt'
```

Comment:
What happens if someone changes the `dw_dnt` key on AuthKeyMap and forget to change this key here? It is best to keep a constant for the string 'dw_dnt` to avoid that mistake

Reply:
I see. I updated the code to use a singular reference to dw_dnt in the Auth class.